src/main/java/io/cdap/e2e/pages/actions/CdfConnectionActions.java

```diff
@@ -366,4 +374,27 @@ public static void verifyConnectionErrorMessage(String errorMessageLocation) {
     AssertionHelper.verifyElementDisplayed(CdfConnectionLocators.locateConnectionFooterError);
     AssertionHelper.verifyElementContainsText(CdfConnectionLocators.locateConnectionFooterError, expectedErrorMessage);
   }
+
+  /**
+   * Click on the Plus Green Button
+   */
+  public static void clickOnAddEntityButton() {
+    clickOnElement(CdfConnectionLocators.plusGreenButton);
+  }
+
+  public static void fixAllIfPrompted() {
+    clickIfDisplayed(CdfConnectionLocators.locatorOfFixAll());
+  }
+
+  /**
+   * Import json Files
+   */
+  public static void importJsonFiles(String s) throws URISyntaxException {
```

Comment:
change the name of ```s``` to something relevant.

Reply:
resolved